### PR TITLE
Better wording when can not load image

### PR DESCRIPTION
**Pull request by victoryforce**
In my experience, I've never seen image files that would be so corrupted that it prevented the decoder from showing them.

I'm sure the vast majority of times a user sees this message is when he/she tries to open an image file from a camera that is not yet supported.

Since the deadline fot string freeze  is so close, I offer my version of this toast message quickly and I'm sure it can be improved. But, IMHO, it is better not to leave the original text.

### CODE REVIEW

**Comment by elstoc** [comment]
New wording is fine with me, but I would still include some text about corrupted images, even if it's not likely.

**Comment by victoryforce** [comment]
@elstoc How about appending these two lines:

if you are sure that the camera model is supported, please consider opening an issue
at https://github.com/darktable-org/darktable

**Comment by elstoc** [comment]
> How about appending these two lines

That seems fine

**Comment by elstoc** [comment]
Have you tried to get this error to display in darktable? I suppose the only question mark now is whether you have time to read it before the message disappears - I can't tell how this particular message is displayed to the user.

**Comment by TurboGit** [comment]
I agree with elstoc, probably too wordy to be read now.

**Comment by victoryforce** [comment]
> Have you tried to get this error to display in darktable? I suppose the only question mark now is whether you have time to read it before the message disappears - I can't tell how this particular message is displayed to the user.

@elstoc Yes, i tested this message. It is displayed for 15 seconds. For me, it's enough time to read it at least twice. :)

**Review by TurboGit — APPROVED** [review]
Ok to go with that and that will be the last modified string for 3.6 !
